index.html

```diff
@@ -30,7 +30,147 @@ <h2 class="connect">LET’S CONNECT</h2>
+                <div class="tags">
+                   <p class="tag">html</p> 
+                   <p class="tag">css</p>
+                   <p class="tag">javascript</p>
+                </div>
+                <div class="button">
+                    <p class="seeproj">See Project</p>
+                </div>
+            </div>
         </section>
-    
+        <section class="main">
+            <div class="container">
+                <h1 class="greet">About <br> Myself</h1>
+            <p class="para"> I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <h2 class="connect">LET’S CONNECT</h2>
+            <div >
+               <ul class="social">
+                    <li> <a href="https://www.linkedin.com/in/daniel-njoroge-3651a2a5/" target="_blank" rel="noopener"><img src="img/Linkedin.png" alt="linkedin" ></a> </li> 
+                    <li> <a href="https://twitter.com/Icharian" target="_blank" rel="noopener"> <img src="img/Twitter.png" alt="twitter"></a></li>    
+                    <li> <a href="https://github.com/Dinicharia" target="_blank" rel="noopener"> <img src="img/github.png" alt="github"></a></li>
+                    <li> <a href="www.angelink.com" target="_blank" rel="noopener"> <img src="img/angela.png" alt="angela"></a></li>
+                    <li> <a href="www.medium.com" target="_blank" rel="noopener"> <img src="img/M.png" alt="m"></a></li>
+                </ul>
+            </div>
+                <div class="button">
+                <p class="getresume">Get My Resume</p>
+                </div>
+                <div class="frame12">
+                    <div class="toolbar">
+                    <h3 class="lang">Languages</h3>
+                    <div class="nbutton"><a href="#"> <img src="img/NButton.png" alt="Normal Button"></a> </div>
+                    </div>
+                    <div class="langlist">
+                        <div class="frame9">
+                            <div class="langicon"><a href="#"> <img src="img/js.png" alt="javascript"></a></div>
+                            <h3 class="lang">JavaScript</h3>
+                        </div>
+                        <div class="frame9">
+                            <div class="langicon"><a href="#"> <img src="img/html.png" alt="html"></a></div>
+                            <h3 class="lang">HTML</h3>
+                        </div>
+                        <div class="frame9">
+                            <div class="langicon"><a href="#"> <img src="img/css.png" alt="css"></a></div>
+                            <h3 class="lang">CSS</h3>
+                        </div>
+
+                    </div>
+                </div>
+                <div class="frame12">
+                    <div class="toolbar">
+                        <h3 class="lang1">Frameworks</h3>
```

Comment:
- [Optional] as demanded by the project requirements, kindly use a nested unordered list to list these categories and technologies. This is best practice.